__tests__/hooks/useFeedHeader.test.js

```diff
@@ -49,14 +49,22 @@ describe('Feed Header should run correctly', () => {
     });
     expect(spyActionFeed).toHaveBeenCalled();
   });
-  it('onBackNormal should run correctly', () => {
+  it('onBackNormal not from feeds should run correctly', () => {
     const {result} = renderHook(() => useFeedHeader({actor, source: 'public'}), {wrapper: Store});
     act(() => {
       result.current.onBackNormalUser();
     });
     expect(mockedGoBack).toHaveBeenCalled();
   });
 
+  it('onBackNormal from feeds should run correctly', () => {
+    const {result} = renderHook(() => useFeedHeader({actor, source: 'public'}), {wrapper: Store});
+    act(() => {
+      result.current.onBackNormalUser(true);
+    });
+    expect(mockedNavigate).toHaveBeenCalled();
+  });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test cases 'onBackNormal not from feeds should run correctly' and 'onBackNormal from feeds should run correctly' seem to be testing the same function `onBackNormalUser()`. However, it's unclear how the boolean argument in line 63 affects the function behavior as there is no difference in the setup of these two tests. Please ensure that the tests are correctly set up to reflect different scenarios or conditions.


<!-- This is an auto-generated comment by OSS CodeRabbit -->